bin/convert-mobile.js

```diff
@@ -66,17 +68,35 @@ function convertIconToXcodeImageSet(iconName, outputRoot) {
 }
 
 const options = yargs
-  .usage('Usage: -n <name of icon, omit if doing bulk>, -o <output path (defaults to ./output)>, -p <target platform (e.g. ios) -insize <16, 24, 32, omit for 16>')
+  .usage('Usage: -n <name of icon, omit if doing bulk>, \n-s <output size, pixels, default 64>, \n-o <output path (defaults to ./output)>, \n-p <target platform (e.g. ios) \n-i <16, 24, 32, omit for 16>')
```

Comment:
I would say it's more appropriate to say the output size in "points" rather than "pixels". 

The point system is essentially the screen real estate afforded to the image on screen, where images with @2x and @3x pixel ratios are used for screens with higher pixel resolution. For screens with higher resolution what we're doing is putting more pixels into the image that takes up the same amount of real space.

Reply:
Also, my understanding is that the number specified by `-s` is equal to the size output at @1x then @2x is calculated by (s * 2) and @3x by (s * 3). Is this understanding correct?